cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -2018,6 +2027,36 @@ fn x64_get_regs(inst: &Inst, collector: &mut RegUsageCollector) {
         | Inst::Fence { .. } => {
             // No registers are used.
         }
+
+        Inst::ElfTlsGetAddr { .. } => {
+            // All caller-saves are clobbered.
+            collector.add_def(Writable::from_reg(regs::rsi()));
```

Comment:
Would it make sense to use the return value of `ABIMachineSpec::get_regs_clobbered_by_call` here?

Reply:
Ah yes, good idea! Done.